### PR TITLE
[FIX] project: fix project privacy_visibility tooltip

### LINKED ISSUE #92734: [14.0] Project: wrong description compared to features
Screenshot from 2022-06-02 14-29-40

Current behavior: **Invited portal and all internal users:** `employees may see everything. Portal users may see project and tasks followed by them or by someone of their company.` In fact, those who use the portal with their company but are not followed will not be able to see this project and task.

Support ticket number submitted via odoo.com/help (optional):

### PR DESCRIPTION
**Pull request by MissingNoShiny**
The project privacy_visibility tooltip doesn't reflect the actual behavior for the "Invited portal users and all internal users" selection.

This PR rephrases the tooltip of that field in order to correctly match its behavior.
It also slightly changes the name of the options to match the new tooltip.

Fixes #92734

Task-2871771

### CODE REVIEW

**Comment by MissingNoShiny** [comment]
Hello @tranngocson1996, I was busy with other stuff this week, but this PR (and another one for helpdesk, since I found out there was the same tooltip issue with helpdesk teams) is now ready for review, so it should hopefully not take too long until it gets merged. :slightly_smiling_face:

**Comment by tranngocson1996** [comment]
> Hello @tranngocson1996, I was busy with other stuff this week, but this PR (and another one for helpdesk, since I found out there was the same tooltip issue with helpdesk teams) is now ready for review, so it should hopefully not take too long until it gets merged. :slightly_smiling_face: 

@MissingNoShiny thank you bro

File: addons/project/i18n/project.pot
Comment (IT-Ideas) [inline addons/project/i18n/project.pot:10]: I am unsure we really need to update those fields... I have just asked @mart-e 
Let's wait for his answer 🙂

File: addons/project/i18n/project.pot
Comment (IT-Ideas) [inline addons/project/i18n/project.pot:97]: Shouldn't we add this in the commit description?

File: addons/project/i18n/project.pot
Comment (IT-Ideas) [inline addons/project/i18n/project.pot:1948]: Ideally, in order to keep the diff minimal, I would not change the place of this traduction in the file.

File: addons/project/i18n/project.pot
Comment (IT-Ideas) [inline addons/project/i18n/project.pot:1403]: Ideally, in order to keep the diff minimal, I would not change the place of this traduction in the file.

**Review by IT-Ideas — COMMENTED** [review]
Thanks 🙂

File: addons/project/i18n/project.pot
Comment (IT-Ideas) [reply addons/project/i18n/project.pot:10]: I have the confirmation that this diff can be removed 🙂

**Review by IT-Ideas — APPROVED** [review]
Thanks 🙂

**Comment by IT-Ideas** [comment]
@robodoo r+

**Comment by IT-Ideas** [comment]
@robodoo r+

File: addons/project/i18n/project.pot
Comment (minhkuheo) [inline addons/project/i18n/project.pot:894]: Would you please tell me what this symbol ">" means in this context?

Comment (IT-Ideas) [reply addons/project/i18n/project.pot:894]: This means we talk about the `the administrator access right level` `of` `project`.

File: addons/project/models/project.py
Comment (Thaotran0967) [inline addons/project/models/project.py:208]: an internal user with no project access rights can't access the task even though the URL is shared and they are part of the followers
@tranngocson1996

Comment (MissingNoShiny) [reply addons/project/models/project.py:208]: Hello, it seems like it doesn't work like that in version 14 indeed, but only in that version, curiously. We will make a fix, thanks for the follow-up.

Comment (sang250399) [inline addons/project/models/project.py:208]: i don't understand this line, can you help me explain? @MissingNoShiny

Comment (MissingNoShiny) [reply addons/project/models/project.py:208]: Hello, it means that if you give the url of the task to an internal user, they can view it regardless of their project access rights. If the project has the visibility "Invited internal users" however, they still need to be part of the followers of the task for this to apply.

Comment (sang250399) [reply addons/project/models/project.py:208]: > Hello, it means that if you give the url of the task to an internal user, they can view it regardless of their project access rights. If the project has the visibility "Invited internal users" however, they still need to be part of the followers of the task for this to apply.

oke, i understand,thanks !

Comment (MissingNoShiny) [reply addons/project/models/project.py:208]: You're welcome :slightly_smiling_face:

**Pull request closed without merging**
